manager/impl/src/main/java/org/jboss/forge/furnace/manager/impl/AddonManagerImpl.java

```diff
@@ -217,12 +217,15 @@ private AddonActionRequest createRequest(final AddonInfo requestedAddonInfo, fin
     */
    private void collectRequiredAddons(AddonInfo addonInfo, List<AddonInfo> addons)
```

Comment:
In that case, we need to distinguish between installing to a specific repository, and just "installing" in general to any repository. The reason is, as I said in chat on Friday, because if you are using an AddonView on your intended installation target repository, then the addon will not be enabled in your AddonView, afaik. This is worth checking, I guess.